README.md

```diff
@@ -15,7 +15,7 @@
 You can clone the repository wherever you want. (I like to keep it in `~/Projects/dotfiles`, with `~/dotfiles` as a symlink.) The bootstrapper script will pull in the latest version and copy the files to your home folder.
 
 ```bash
-git clone git@github.com:DaliaResearch/dotfiles.git && cd dotfiles && source bootstrap.sh
+git clone http://git@github.com/DaliaResearch/dotfiles.git && cd dotfiles && source bootstrap.sh
```

Comment:
@adaline Why this is better?

Reply:
Someone can follow these instructions without having to be logged into GitHub, for example - a new employee setting up a new computer ;)